cms/djangoapps/contentstore/xblock_storage_handlers/xblock_helpers.py

```diff
@@ -16,11 +16,17 @@ def usage_key_with_run(usage_key_string):
     return usage_key
 
 
-def get_children_tags_count(xblock):
+def get_children_tags_count(xblock, get_parent=False):
```

Comment:
Hmm.. this addition makes this method read really weirdly. It's just a helper method, not an API (and our call is the only one using it), so I think it should have this signature this instead (with a little refactoring to make that work as described):

```suggestion
def get_tags_count(xblock, include_children=False):
```

Reply:
Updated: https://github.com/openedx/edx-platform/pull/36318/commits/7992bcf5492894e4003f0bb8db0a4078a4968414